manifests/deploy/ocp-testing/olm/bundle/4.3/meteringconfig.crd.yaml

```diff
@@ -1548,6 +1548,29 @@ spec:
                                   properties:
                                     useInstanceCredentials:
                                       type: boolean
+                            prometheus:
+                              type: object
+                              properties:
+                                enabled:
+                                  type: boolean
+                                uri:
+                                  description: URI for Prometheus.
+                                  type: string
+                                chunkSizeDuration:
+                                  description: Default size of each Prometheus query chunk.
+                                  type: string
+                                maxQueryRangeDuration:
+                                  description: Default range for Prometheus query, divided into chunkSizeDuration chunks.
+                                  type: string
+                                cacheDuration:
+                                  description: Time to live for Presto checking connector properties values.
+                                  type: string
+                                bearer:
+                                  type: object
+                                  properties:
+                                    tokenFile:
```

Comment:
@chancez do you think this should just `bearerTokenFile` for the time being, or do you think there will be more authentication/bearer configurations in the future that warrants it's own section?

Reply:
Good question. I guess generally speaking, being able to specify a path directly isn't very useful actually since it'll be coming either from the service account secret, or an arbitrary user supplied secret.

In the case of a user supplied secret, they wouldn't likely have much choice in the location of the file, and we would just mount it to a known location.

So perhaps we should have an. A `useServiceAccountToken` bool option and eventually (not now) we could add a `secretName` config for supplying your own credentials.

WDYT?

Reply:
Good question. I guess generally speaking, being able to specify a path directly isn't very useful actually since it'll be coming either from the service account secret, or an arbitrary user supplied secret.

In the case of a user supplied secret, they wouldn't likely have much choice in the location of the file, and we would just mount it to a known location.

So perhaps we should have an. A `useServiceAccountToken` bool option and eventually we could add a `secretName` config for supplying your own credentials.

Reply:
Copied this comment set to https://github.com/operator-framework/operator-metering/pull/1017 comments.